tests/integration/it_table_shapes_test.py

```diff
@@ -26,6 +36,30 @@ def test_ordered_list(client):
     ).status_code == HTTPStatus.OK
 
 
+def test_filtered_list(client, db_session):
+    db_session.add(TableShape(id=1, description="B", picture="pic"))
```

Comment:
@ekondrashev if you are using the same set of table shapes, mb move them to a fixture?

Reply:
@vryazanov it is not always same. Additionally I'd say that current sets will change once custom `Like` filter is in place, so it is a bit early to define same sets to be moved to a fixture.